database/java/2017/12629.yaml

```diff
@@ -0,0 +1,21 @@
+cve: 2017-12629
+title: "Apache Solr config API add-listener command is vulnerable to Remote Code Execution attacks"
+description: >
+    Remote code execution occurs in Apache Solr before 7.1 with Apache Lucene before 7.1 by exploiting XXE in conjunction with use of a Config API add-listener command to reach the RunExecutableListener class.
+cvss_v2: 7.5
+references:
+    - https://github.com/AsyncHttpClient/async-http-client/issues/1455
+    - https://nvd.nist.gov/vuln/detail/CVE-2017-12629
+    - http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-12629
+    - http://www.cvedetails.com/cve/CVE-2017-12629/
+    - http://lucene.472066.n3.nabble.com/Re-Several-critical-vulnerabilities-discovered-in-Apache-Solr-XXE-amp-RCE-tt4358355.html
+affected:
+    - groupId: "org.apache.solr"
+      artifactId: "solr-core"
+      version:
+        - "<=7.0.1"
+        - ">=5.5.0"
+      fixedin:
+        - "==5.5.5"
+        - "==6.6.2"
```

Comment:
I'm wondering why we use `==` here instead of `>=` ?

Reply:
6.6.2 and 7.1.0 are patched with the fix, but 7.0.0,7.0.1 are vulnerable
So cant specify >= 6.6.2

Makes sense?

Reply:
Yes, thank you!